src/Swagger/Serializer/DocumentationNormalizer.php

```diff
@@ -41,15 +42,17 @@
     private $resourceClassResolver;
     private $operationMethodResolver;
     private $operationPathResolver;
+    private $requestContext;
 
-    public function __construct(ResourceMetadataFactoryInterface $resourceMetadataFactory, PropertyNameCollectionFactoryInterface $propertyNameCollectionFactory, PropertyMetadataFactoryInterface $propertyMetadataFactory, ResourceClassResolverInterface $resourceClassResolver, OperationMethodResolverInterface $operationMethodResolver, OperationPathResolverInterface $operationPathResolver)
+    public function __construct(ResourceMetadataFactoryInterface $resourceMetadataFactory, PropertyNameCollectionFactoryInterface $propertyNameCollectionFactory, PropertyMetadataFactoryInterface $propertyMetadataFactory, ResourceClassResolverInterface $resourceClassResolver, OperationMethodResolverInterface $operationMethodResolver, OperationPathResolverInterface $operationPathResolver, RequestContext $requestContext)
```

Comment:
Or we can do this in the `RouterOperationPathResolver` but then resolvers result won't be consistent anymore.